core/src/main/java/com/datastax/oss/driver/internal/core/util/Strings.java

```diff
@@ -270,48 +269,54 @@ private Strings() {}
           "by",
           "columnfamily",
           "create",
+          "default",
```

Comment:
I don't know if you used that to create this list, but there is a constant field in `org.apache.cassandra.cql3.ReservedKeywords#reservedKeywords` with the list of reserved keywords. I manually validated that our list matches theirs for the current state of the `trunk` branch.